### PR TITLE
make shutdown timeout configurable on builder

### PR DESCRIPTION
**Pull request by derelbenkoenig**
Y'all already know what it is

### CODE REVIEW

File: src/test/java/com/bandwidth/sqs/consumer/SqsConsumerTest.java [hunk 1]
```diff
@@ -99,6 +99,7 @@ public SqsConsumerTest() {
                 .withBackoffStrategy(backoffStrategyMock)
                 .withExpirationStrategy(expirationStrategyMock)
                 .withAutoExpire(true)
+                .withShutdownTimeout(SqsConsumer.DEFAULT_SHUTDOWN_TIMEOUT)
```
Comment (fuchsnj) [inline src/test/java/com/bandwidth/sqs/consumer/SqsConsumerTest.java]: Why is this needed? Doesn't it use the default... by default?

Comment (derelbenkoenig) [reply src/test/java/com/bandwidth/sqs/consumer/SqsConsumerTest.java]: Coverage
